docs/quick_start.md

```diff
@@ -16,64 +16,77 @@ limitations under the License.
 
 # Quick Start
 
-The goal of this section is to analyze a simple libtorch model.
-
-1. Build Model Anlayzer Container:
+The steps below will guide you through using model analyzer to analyze a simple PyTorch model. If you are not using the docker installation, you may skip the first step. The instructions below assume a directory structure like the following:
```

Comment:
It would be great if this doc can be modified to suggest using the SDK container.